modules/n1ql/pages/n1ql-language-reference/subquery-examples.adoc

```diff
@@ -12,8 +12,7 @@ Note that the subquery uses nested paths in the FROM clause.
 ----
```

Comment:
First, please add the following admonition _before_ the first Example heading:

``` asciidoc
WARNING: Please note that the examples below will alter the data in your sample buckets.
To restore your sample data, remove and reinstall the `travel-sample` bucket.
Refer to xref:manage:manage-settings/install-sample-buckets.adoc[Sample Buckets] for details.
```

Secondly, please change the language for the first code block to N1QL:

``` asciidoc
[source,n1ql]
```

Reply:
This warning applies to all the examples, not just example 1, so please move the warning to line 3 — _before_ the first Example heading